recipes/libvips/meta.yaml

```diff
@@ -0,0 +1,132 @@
+# Notes
+# 
+# - `openslide` support is missing, since that's in bioconda and this is a
+#   conda-forge recipe. I suppose we'd need to move to bioconda, or ask
+#   for `openslide` to move to conda-forge.
+#
+# - `lcms`, `libexif` support is missing.  EXIF metadata and colour management
+#   is important. I guess we shuld make conda-forge packages for them too.
+#
+# - `libgsf-1` is missing, it means no deepzoom output. Again, we'd need to
+#   package it.
+#
+# - `niftiio` is missing, I guess this is in bioconda as well.
+#
+# - `openexr` is missing.
+# 
+# - `libheif` is missing, so no HEIC support. Again, we'd need to package
+#   that separately.
+# 
+# - `orc` is missing, so there's no SIMD support. Things like convolution
+#   will be up to about 3x slower.
+# 
+# - `imagemagick` is built with `jpeg`, not `libjpeg-turbo`, so we must use
+#   `jpeg` too. This is very unfortunate. It would be good if IM could move
+#   to turbo as well.
+# 
+# - I had to pin pango to 1.40.14 to get librsvg and imagemagick to go on
+#   together.
+# 
+# - libvips is marked up for autovectorisation by gcc. This is enabled at -O3
+#   and above.
+
+{% set name = "libvips" %}
+{% set version = "8.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libvips/libvips/releases/download/v{{ version }}/vips-{{ version }}.tar.gz
+  sha256: a0ee255a2a1ebfea5b2dff2a780824d7157a78c010d7ddd531279aacefbf2539 
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config        # [not win]
+  host:
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - hdf5
+    - imagemagick
+    - jpeg
+    - libmatio
+    - libpng
+    - librsvg 
+    - libtiff
+    - libwebp
+    - pango =1.40.14
+    - poppler
+    - zlib
+  run:
+    - bzip2 
+    - cairo 
+    - cfitsio 
+    - expat
+    - fftw
+    - fontconfig
+    - freetype
+    - gdk-pixbuf
+    - giflib
+    - glib
+    - hdf5
+    - imagemagick
+    - jpeg
```

Comment:
The only thing I can add is that conda forge has been adding run exports to many of these libraries so technically you only have to include them in the gostsection

Reply:
The issue, is that the lintner won't help you here, so you have to go through them manually. This is probably a good feature request for the linter